files/mcloud/node_modules/@appium/base-driver/build/lib/protocol/protocol.js

```diff
@@ -393,128 +393,131 @@ function buildHandler(app, method, path, spec, driver, isSessCmd) {
         }
         catch (err) {
 
-            async function updateDeviceStatus(token, udid, note) {
+            // token: string - auth token from stf 
+            // udid: string -  unique id of device 
+            // note: string - comment with error for device which would shows in stf
+            // isMakedDeviceUnhealthy: boolean - if true device status will change to Unhealthy, if false comment will added 
```

Comment:
let's make function more robust providing possibility to change into any state. also rename function to `updateDevice`